packages/components/dropdown/src/dropdown-item-impl.vue

```diff
@@ -3,7 +3,6 @@
     v-if="divided"
     role="separator"
     :class="ns.bem('menu', 'item', 'divided')"
-    v-bind="$attrs"
```

Comment:
Maybe we shouldn't delete directly, but just exclude the `id`.

Reply:
It stands to reason that this would lead to destructive updates. But is it reasonable for `separator` and `menuiitem` to bind the same attributes?

Reply:
It has been so long that it is hard to clearly understand the considerations at that time. However, I personally think that binding the same attributes may facilitate the clear display of the grouping relationship between the item and the corresponding divider?

Reply:
I have an idea: extend `divided` to support `Object` type for attribute configuration. Is it possible?

Reply:
To me, this seems feasible, but we could probably get more opinions.  cc @element-plus/backers

Reply:
Removing this line looks good to me. I noticed [the test case](https://github.com/element-plus/element-plus/pull/5779/files#diff-fb8a1dec280aaa963d1509eac461358cc305563bdf74d743c58401ad6ee5745aR491) didn't cover this line of code, so the author might add the code no particular reason.

BTW, I can't receive a notification when you mentioned backers. @btea

Reply:
I also support deleting this line of code, and binding attributes to `item` and `divided` feels strange